temboardagent/inventory.py

```diff
@@ -84,7 +84,10 @@ def os_flavor(self):
 
     def linux_distribution(self):
         if self.os == 'Linux':
-            return " ".join(platform.linux_distribution()).strip()
+            try:
+                return " ".join(platform.linux_distribution()).strip()
+            except AttributeError:
+                return 'Distribution info unavailable'
```

Comment:
It would be clearer to me if we explicitly check for the python version (also better for future reference) rather than catching `AttributeError`.